about2.html

```diff
@@ -0,0 +1,133 @@
+                    Vivamus
+                    sit amet nisi a tellus interdum placerat vitae sit amet dolor. Praesent placerat hendrerit mi, in
+                    ultrices lectus vulputate et. Aenean lacinia feugiat ultrices. Nullam dapibus tortor at purus
+                    eleifend
+                    bibendum. Etiam tristique, quam eleifend auctor bibendum, nunc massa condimentum odio, et facilisis
+                    elit
+                    odio in felis. Vivamus tincidunt est tempus orci tempus, non cursus ex maximus. Pellentesque
+                    habitant
+                    morbi tristique senectus et netus et malesuada fames ac turpis egestas.
+
+                    Fusce at dui a mauris sollicitudin accumsan. Aenean gravida eget metus ut porta. Ut at varius eros.
+                    Aenean ac lectus molestie, varius ante at, faucibus dolor. Sed convallis lectus turpis, ac egestas
+                    libero accumsan ac. Vivamus vulputate massa nec sapien cursus facilisis. Quisque turpis diam,
+                    porttitor
+                    sed metus a, sodales placerat sem. Duis nec ipsum et ipsum lobortis lobortis. Fusce euismod blandit
+                    neque non accumsan. Sed vulputate venenatis nibh, et sagittis sem pretium id. Nulla at dapibus
+                    velit.
+                    Proin nec nisi porttitor, egestas sapien a, elementum.
+                </div>
+
+                <p>2022.02.18(SAT)- 20(MON) </p>
+                <p>Gran Salon Corferias, Bogotá-Colombia</p>
+            </div>
+
+        </section>
+        <section class="news">
+            <div class="container program_titles" >
+                <h4><strong>Creating High-Performance Teams Conference Bogotá Logo</strong></h4>
+                <hr>
+
+                <p>The Bogota brand, approved by the Council, has identified the capital over the last decade and was
+                    created in 2008 with the participation of the Mayor's Office of Bogota, through the District
+                    Institute
+                    of
+                    Tourism, the IDT, the Bogota Chamber of Commerce, the CCB, and Invest Bogota, with the vision of
+                    creating a
+                    city strategy that would position the city as a tourist destination, investment and
+                    business articulation at national and international level.
+                </p>
+                <img src="./images/logo_bogota.jpg" alt="logo" />
+                <h4><strong>See the past CHPT</strong></h4>
+                <hr>
+                <h4><strong>Take a look at the latest conferences in Brazil and Huakanda.</strong></h4>
+                <div class="past_events">
+                    <img src="./images/past_event.png" alt="logo" />
+                    <img src="./images/past_event_two.png" alt="logo" />
+                </div>
```

Comment:
- [ ] As the previous reviewer has pointed out, in other to make this look like the original design, please ensure that the cards are displayed in a row instead of a column as it is in the original design.
- [ ] There should also be some details(year, etc) on the image card.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169634958-0fbe68a9-ed4d-49ec-bea9-3bd95efb2b2d.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/169634976-91532c02-489f-4fed-b8da-8e02409d5fcb.png)